modules/swagger-codegen/src/main/resources/csharp/Project.mustache

```diff
@@ -88,7 +88,10 @@
     {{/netStandard}}
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="**\*.cs"/>
+    <Compile Include="Api\**\*.cs"/>
```

Comment:
@wing328 sounds good. I'm going to check in Rider and Visual Studio 2015.

@voydz It sounds like you're experiencing a bug in Xamarin. I know prior to 6.0, there were issues with Build and Rebuild not executing cleanup tasks. I'll see if we can exclude `obj` from the glob in a way that works in Xamarin.